bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/seqcomp/GhostSeqRepeatExp.java

```diff
@@ -0,0 +1,93 @@
+package edu.gemini.spModel.seqcomp;
+
+import edu.gemini.pot.sp.SPComponentType;
+import edu.gemini.spModel.gemini.ghost.Ghost$;
+import edu.gemini.spModel.obsclass.ObsClass;
+import edu.gemini.spModel.obscomp.InstConstants;
+import edu.gemini.spModel.pio.ParamSet;
+import edu.gemini.spModel.pio.Pio;
+import edu.gemini.spModel.pio.PioFactory;
+
+import java.io.Serializable;
+
+/**
+ * A SeqRepeat specifically for GHOST: no coadds, two exposures.
+ */
+public class GhostSeqRepeatExp extends SeqRepeat implements Serializable {
```

Comment:
Is this the observe replacement?  `SeqRepeat` is pretty generic.  Should this one be `final` I think?

Reply:
This one is extended by:
`GhostSeqRepeatDarkObs`
and
`GhostSeqRepeatFlatObs`.

Reply:
It adds the detectors to `SeqRepeat`, which is the superclass of `GhostSeqRepeatExp` and `SeqRepeatCoaddExp`.